src/Nethermind/Nethermind.Runner/Ethereum/Steps/RegisterRpcModulesClique.cs

```diff
@@ -0,0 +1,41 @@
+﻿//  Copyright (c) 2018 Demerzel Solutions Limited
+//  This file is part of the Nethermind library.
+// 
+//  The Nethermind library is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+// 
+//  The Nethermind library is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//  GNU Lesser General Public License for more details.
+// 
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
+
+using System.Threading.Tasks;
+using Nethermind.Clique;
+using Nethermind.JsonRpc.Modules;
+using Nethermind.Runner.Ethereum.Context;
+
+namespace Nethermind.Runner.Ethereum.Steps
+{
+    public class RegisterRpcModulesClique : RegisterRpcModules
+    {
+        private readonly CliqueEthereumRunnerContext _context;
```

Comment:
same for Clique steps in Clique namespace if possible

Reply:
by namespace I mean -moving it to a Clique specific project and leaving the main project open

Reply:
same as above